### PR TITLE
Fix source file for pre-installed riscv toolchain

### PR DESCRIPTION
**Pull request by cathylu10**

### CODE REVIEW

File: fast-setup.sh [hunk 1]
```diff
@@ -88,7 +88,9 @@ then
 fi
 
 # update source.sh
-echo "export RISCV=$(pwd)/riscv${BITS}" > ./source.sh
+GCC_PATH=$(whereis riscv$BITS-unknown-linux-gnu-gcc)
```
Comment (dayeol) [inline fast-setup.sh:91]: can you use `which` instead so that we can remove `cut` command below?
e.g., GCC_PATH=$(which riscv$BITS-unknown-linux-gnu-gcc)

**Review by dayeol — CHANGES_REQUESTED** [review]

**Comment by dayeol** [comment]
@cathylu10 could you change this?

File: fast-setup.sh (hunk 1 above)
Comment (cathylu10) [reply fast-setup.sh]: Fixed

**Review by dayeol — APPROVED** [review]
lgtm

**Pull request merged**